oidc-js/src/worker/web-worker-client.ts

```diff
@@ -98,7 +98,7 @@ import { getAuthorizationCode } from "../utils";
  *
  * ```
  */
-export const WebWorkerClient: WebWorkerSingletonClientInterface = (function(): WebWorkerSingletonClientInterface {
+export const WebWorkerClient: WebWorkerSingletonClientInterface = (function (): WebWorkerSingletonClientInterface {
```

Comment:
Can we convert to an arrow function here ?

Reply:
Resolved by dc31070